src/frontend/csrs-portal/README.md

```diff
@@ -12,7 +12,7 @@ The following list includes the required software needed to run the application,
 
 [Download](https://git-scm.com/downloads) and install the Git version control system, and optionally [download](https://www.gitkraken.com) and install the free GitKraken Git GUI client.
 
-Clone the Traffic Court Online repository into a project directory GitKraken or the terminal by typing:
+Clone the CSRS repository into a project directory GitKraken or the terminal by typing:
```

Comment:
Readme does not need to be so granular to specify a Git browser or how to clone.  Can simply state clone repo. 
However as this is a public repo it should be forked